### PR TITLE
Try upgrading minikube for k8s 1.18

### PR DESCRIPTION
**Pull request by foot**
Fixes health checker running after 1.18 was released yesterday

Minikube 1.7.3 adds some conntrack stuff needed by 1.18 https://github.com/kubernetes/minikube/releases

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -124,6 +124,8 @@ jobs:
         run:
           name: Install integration test dependencies
           command: |
+            sudo apt-get update -y
```
Comment (bboreham) [inline .circleci/config.yml]: I know this is traditional when you do it on your desktop, but is it necessary here?
Using whatever metadata is on the image seems preferable, for consistency of test runs.  If that works.

Comment (foot) [reply .circleci/config.yml]: Yes! The apt-get cache seems to be completely empty when the vm spins up so if you skip this it says "cannot find package conntrack" etc.

Comment (foot) [reply .circleci/config.yml]: Most of the time in CircleCI. I have not actually tried it in this case. Lets give it a quick test..

Comment (foot) [reply .circleci/config.yml]: It did not work, gotta update

Comment (bboreham) [reply .circleci/config.yml]: Ah, thanks for trying.

**Pull request merged**